package WebGame.Server.Stock;
message HeartBeat {
  enum traits { value = 1 ;}
}
